main.py

```diff
@@ -39,10 +39,10 @@
 app.include_router(utility.router, prefix=prefix)
 
 
-@app.on_event("startup")
-def on_startup():
-    """On server startup, seed the database."""
-    db = SessionLocal()
-    uc = UtilityController(db);
-    seed(uc)
-    db.close()
+# @app.on_event("startup")
```

Comment:
Is there any specific reason for leaving this commented code here?

Reply:
Yes in case of needing to re-seed the database again on startup, but i can remove it.